.github/workflows/publish.yml

```diff
@@ -5,7 +5,7 @@ on:
       - master
 jobs: 
   publish:
-    if: "!startsWith(github.event.head_commit.message, 'ci:|docs:|')"
+    if: "!startsWith(github.event.commits[0].message, 'ci:|docs:|test:|')"
```

Comment:
This makes it so we only republish on code changes?

Reply:
That's what I'm hoping, but `event.head_commit` didn't work. In another terraform report I use `event.commit[0]` so I'm giving that a chance... although this is a `fix` so it won't apply anyways 🤷‍♀️
What I don't know is in the `event.commit` is available on PR merge....